websites/docs/intro/installation.ko.md

```diff
@@ -11,18 +11,18 @@ title: 설치하기
 npm install @suspensive/react
 ```
 
-### yarn
+### pnpm
 
-혹은 <a href="https://classic.yarnpkg.com/en/docs/install/" target="_blank">yarn</a>을 사용한다면:
+혹은 <a href="https://pnpm.io/" target="_blank">pnpm</a>을 사용한다면:
 
 ```shell
-yarn add @suspensive/react
+pnpm install @suspensive/react
```

Comment:
```suggestion
pnpm add @suspensive/react
```

Reply:
Since I only changed the location of the existing code, I didn't think of any grammatical errors. Thanks @manudeli

Reply:
Cool, It's my fault👍